classes/class-wc-connect-nux.php

```diff
@@ -1,5 +1,16 @@
 <?php
 
+if ( ! class_exists( 'Jetpack_Install_Status' ) ) {
```

Comment:
Even without the abstract keyword you don't have to create an instance to access the constants. They are basically static variables on the class name:

```
class test_class {	const VAL = 'hello'; }
echo test_class::VAL;
```

Abstract is used to define an "incomplete" parent class, whose abstract functions must be defined in derived classes for them to be non-abstract. It's for that reason that abstract classes are impossible to instantiate.